pymc/func_utils.py

```diff
@@ -36,7 +36,7 @@ def find_constrained_prior(
 ) -> Dict[str, float]:
     """
     Find optimal parameters to get `mass` % of probability
-    of `pm_dist` between `lower` and `upper`.
+    of a `pm_dist` between `lower` and `upper`.
```

Comment:
```suggestion
    of a :ref:`distribution <api_distributions>` between `lower` and `upper`.
    
```

The distribution change is so the text shows distribution but rendered as a link to the distributions api page. The empty line is needed to comply with numpydoc: https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/docstring_tutorial.html#short-summary preferably one line, can't be more than one long sentence though and definotely not a whole paragraph with a note too